pallets/automation-time/src/mock.rs

```diff
@@ -15,8 +15,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use core::cell::RefCell;
-
```

Comment:
A lot of 🔴 below because XCMP pallets and functionality were moved to xcmp-handler.